judging/scripts/run.sh

```diff
@@ -9,14 +13,17 @@ set -o nounset
 set -o pipefail
 set -o xtrace
 
+# What mode is this sandbox running in? "prescriptions" or "predictions"
+mode="prescriptions"
```

Comment:
This will vary per type of sandbox. Except...we won't update this file on the Phase 1 sandboxes so it will only ever be deployed to the Phase 2 sandboxes where it will have the value "prescriptions".